tests/test_sqlcreds.py

```diff
@@ -389,3 +409,29 @@ def test_sqlcreds_connection_from_sqlalchemy(sql_creds):
     df = pd.read_sql(con=test_engine, sql="SELECT TOP 1 * FROM sys.objects")
 
     assert df.shape[0] == 1
+
+
+@pytest.mark.usefixtures("database")
+def test_sql_creds_connection_for_odbc_kwargs_encrypt(sql_creds):
+    """
+    Tests that the SqlCreds object can be created
+
+    * Without Username and Password (Windows Auth)
+    * Use blank port
+    """
+    creds = SqlCreds(
+        server=sql_creds.server,
+        database=sql_creds.database,
+        username=sql_creds.username,
+        password=sql_creds.password,
+        driver_version=sql_creds.driver_version,
+        port=sql_creds.port,
+        odbc_kwargs=dict(Encrypt="no"),
+    )
+    assert str(creds.engine.url) == _quote_engine_url(
+        "Driver={ODBC Driver 99 for SQL Server};Server=tcp:test_server,1433;Database=test_database;UID=me;PWD=secret;Encrypt=no"
```

Comment:
Why does `ODBC Driver 99` work?

Reply:
It does not. I just couldn't get the local test environment up and running. It would be nice if that would work simply with a docker-compose call and matching credentials for Linux.